src/main/groovy/io/micronaut/docs/dropdown/CreateReleaseDropDownTask.groovy

```diff
@@ -1,72 +0,0 @@
-package io.micronaut.docs.dropdown
-
-import groovy.json.JsonSlurper
-import groovy.transform.CompileDynamic
-import groovy.transform.CompileStatic
-import io.micronaut.build.utils.GithubApiUtils
-import org.gradle.api.DefaultTask
-import org.gradle.api.tasks.Input
-import org.gradle.api.tasks.OutputFile
-import org.gradle.api.tasks.TaskAction
-
-@CompileStatic
-class CreateReleasesDropdownTask extends DefaultTask {
```

Comment:
is this file deletion intentional?

Reply:
Yes, this file was converted to Java long ago, and apparently I had forgotten to delete the Groovy version